main.go

```diff
@@ -378,10 +378,17 @@ func main() {
 
 		os.Exit(task(token))
 	} else if cmd == "web" {
+		var directory string = "~/.oauth2l"
```

Comment:
Let's simplify the if statements by updating "directory" with opts.Web.Directory once at the very beginning if not empty, then use "directory" for the rest of this section.